src/main/java/com/bzabek/ticketbookingapp/controller/ReservationController.java

```diff
@@ -0,0 +1,49 @@
+package com.bzabek.ticketbookingapp.controller;
+
+import com.bzabek.ticketbookingapp.entities.Screening;
+import com.bzabek.ticketbookingapp.entities.SeatReserved;
+import com.bzabek.ticketbookingapp.model.MakeReservation;
+import com.bzabek.ticketbookingapp.model.ReservationCreated;
+import com.bzabek.ticketbookingapp.service.ReservationService;
+import com.bzabek.ticketbookingapp.service.ScreeningService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("reservation")
+public class ReservationController {
+
+    private ReservationService reservationService;
+    private ScreeningService screeningService;
+
+    public ReservationController(ReservationService reservationService, ScreeningService screeningService) {
+        this.reservationService = reservationService;
+        this.screeningService = screeningService;
+    }
+
+
+    @GetMapping(path = "/screenings")
+    public ResponseEntity<List<Screening>> getAllScreenings(@RequestParam String from, @RequestParam String to){
```

Comment:
from i to to string - ale jaki one mają format? czy ktoś to sprawdza lub wymusza?

Reply:
Nie sprawdzam formatu. Można określić czas (timestamp) w dowolny sposób.